voucher/utils.py

```diff
@@ -95,7 +103,10 @@ def get_eligible_coupon_choices(voucher):
             json.dumps(
                 (course_matches[i].product.first().id, valid_coupons[i].coupon.id)
             ),
-            course_matches[i].title,
+            "{title} - {start_date}".format(
+                title=course_matches[i].title,
+                start_date=course_matches[i].start_date.date(),
```

Comment:
If the start date is None, this will throw an exception.  Start dates shouldn't be None, but it might happen due to human input error and the model allows it.

Reply:
this always confuses me, why we want a course live without start date? can we make this field required ?
CC @pdpinch

Reply:
I'd like to keep this moving. Is it OK to leave this as is and open another issue to validate start dates?

Reply:
Fine with me